python/nano/src/bigdl/nano/deps/inference/model.py

```diff
@@ -0,0 +1,123 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from typing import List, Tuple
+from bigdl.nano.pytorch.lightning import LightningModuleFromTorch
+import inspect
+from torch.utils.data import DataLoader
+import torch
+
+
+class AcceleratedLightningModule(LightningModuleFromTorch):
```

Comment:
This should be a generic class and not in `deps` module?

Reply:
Yes, this class doesn't introduce any new module. But it intends to be a base class of onnx, openvino, inc model.  
So if I put them in pytorch folder, then onnx, openvino, inc will call `bigdl.nano.pytorch.xxx` and be called by `bigdl.nano.pytorch.trainer`, then their relation is a bit confusing in my perspective.